sw/device/silicon_creator/manuf/BUILD

```diff
@@ -0,0 +1,31 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules:opentitan_test.bzl", "cw310_params", "opentitan_functest", "verilator_params")
+
+opentitan_functest(
+    name = "ft",
```

Comment:
nit: maybe a more descriptive name for the test? (can update later too)

Reply:
should I move this under `manuf/lib/provisioning_functest.c`?

Reply:
sgtm, that would be consistent with Si creator code

Reply:
Done.

Reply:
Done.